slack_sdk/web/client.py

```diff
@@ -1477,6 +1477,59 @@ def apps_event_authorizations_list(
         )
         return self.api_call("apps.event.authorizations.list", params=kwargs)
 
+    def apps_notifications_subscriptions_create(
+        self,
+        *,
+        trigger_id: str,
+        channel_id: str,
+        name: str,
+        type: Optional[Dict] = None,
+        resource_link: Optional[str] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Create a new notification subscription."""
```

Comment:
I know the URL is not yet publicly available but can you add `https://api.slack.com/methods/apps.notifications.subscriptions.create` for  consistency?